server/routes/changeEstablishment.ts

```diff
@@ -93,6 +93,7 @@ async function getAvailablePrisonsForUser(
   Object.keys(supportedPrisons)
     .filter(prisonId => userCaseLoadsIds.includes(prisonId))
     .forEach(prisonId => {
+      // @ts-ignore
```

Comment:
You can remove the `@ts-ignore` and fix the type error in this file by changing line 91 to:
```typescript
const availablePrisonsForUser: Record<string, string> = {}
```